cloudbuild-publish.yaml

```diff
@@ -0,0 +1,33 @@
+steps:
+  - name: 'gcr.io/cloud-builders/gcloud'
+    script: |
+      #!/usr/bin/env bash
+      set -e
+      REPO_TOKEN="$(gcloud auth print-access-token)"
+      cat > ~/.npmrc <<EOL
+      //registry.npmjs.org/:_authToken=$NPM_TOKEN"
+      registry=https://us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/
+      //us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/:always-auth=true
+      //us-npm.pkg.dev/artifact-foundry-prod/ah-3p-staging-npm/:_authToken=${REPO_TOKEN}
+      EOL
```

Comment:
The `npmrc` file can be a real file rather than doing this, and you can specify to use the one in the repo `--userconfig=./.npmrc`. This avoids having to cat contents of files as part of build scripts, which is a bit weird

Reply:
I tried having a local one. somehow the `bazel yarn install` didn`t work with it. It was a bit weird because `yarn config ` showed the correct information.

Reply:
Moreover, this is the method suggested by Airlock team.

Reply:
We're going to swap to `rules_js` soon, it will probably work then. Let's update it at that point.